CHANGELOG.md

```diff
@@ -0,0 +1,50 @@
+<a name="1.115.0"></a>
```

Comment:
As we're starting from scratch / doing it manually, is it okay if we fix something that's always bugged me in the other repos? These anchors are completely redundant, GitHub does it all automagically from the heading anyway.

Reply:
Oh nvm, it's generated I notice.